src/templates/_macros/app_tile.html

```diff
@@ -68,7 +68,7 @@
         'slug': app.slug
       }) }}
     {% endif %}
-    {% set notices = app_notices(app) %}
+    {% set notices =  [] if app.loading == true else app_notices(app) %}
```

Comment:
No need to compare to `true` just use: `[] if app.loading else app_notices(app)`.